content-src/styles/_theme.scss

```diff
@@ -82,7 +82,7 @@ body {
     // General styles
     --newtab-background-color: #{$grey-80};
     --newtab-border-primary-color: #{$grey-10-80};
-    --newtab-border-secondary-color: #{$grey-10-10};
+    --newtab-border-secondary-color: #{$grey-60};
```

Comment:
Is this something that would change non discovery stream dark mode?

Reply:
Good call. I can tweak that so it's separate.